agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.h

```diff
@@ -82,6 +82,15 @@ class monitor_thread : public beerocks::socket_thread {
      */
     bool create_ap_metrics_response(uint16_t mid, const std::vector<sMacAddr> &bssid_list);
 
+    /**
+     * @brief Creates Beacon Metrics Response message
+     *
+     * @param mid Message ID.
+     * @param hal_data Data received from hostapd.
+     * @return True on success and false otherwise.
```

Comment:
```suggestion
     * @return true on success, false otherwise.
```
no need to capitalized reserved keywords